crazyflie_sim/crazyflie_sim/crazyflie_sil.py

```diff
@@ -81,14 +81,17 @@ def __init__(self, name, initialPosition, controller_name, time_func):
         self.motors_thrust_uncapped = firm.motors_thrust_uncapped_t()
         self.motors_thrust_pwm = firm.motors_thrust_pwm_t()
 
+        self.mellinger_control = firm.controllerMellinger_t()
```

Comment:
I suggest moving that to before line 94 (as part of the elif).

Reply:
@whoenig rights! thank you for this, this would prevent creating a redundant variable, this is addressed in the next commit https://github.com/IMRCLab/crazyswarm2/pull/171/commits/df6d146e416973daae6988b710abfe315b3c9462